playground.py

```diff
@@ -50,8 +51,12 @@
 # provision our data sources (knowledgebase is provided by default
 # but initialized here for debug)
 kb = KnowledgeBase(context)
-map = Map(GOOGLE_MAPS_KEY)
-providers = [map]
+providers = []
+
+# If we have a Google Maps API key, we can use it as a provider, too
+if GOOGLE_MAPS_KEY:
```

Comment:
LLM problem: having a single provider I think breaks the few-shot classification. Can you confirm running simon without another provider (even a dummy provider should do) won't break?